.travis.yml

```diff
@@ -6,27 +6,14 @@ otp_release:
   - 18.2
   - 18.3
   - 19.0
-sudo: false
-addons:
-  apt:
-    packages:
-    - automake
-    - bison
-    - flex
-    - g++
-    - libboost-dev
-    - libboost-test-dev
-    - libboost-program-options-dev
-    - libevent-dev
-    - libtool
-    - libssl-dev
-    - pkg-config
+sudo: required
```

Comment:
In general, it depends on how you have docker set up.  Travis requires it: https://docs.travis-ci.com/user/docker/